build.sh

```diff
@@ -166,11 +166,35 @@ __rootbinpath="$__scriptpath/bin"
 __msbuildpackageid="Microsoft.Build.Mono.Debug"
 __msbuildpackageversion="14.1.0.0-prerelease"
 __msbuildpath=$__packageroot/$__msbuildpackageid.$__msbuildpackageversion/lib/MSBuild.exe
-__BuildArch=x64
 __buildmanaged=false
 __buildnative=false
 __TestNugetRuntimeId=win7-x64
 
+# Use uname to determine what the CPU is.
+CPUName=$(uname -p)
+case $CPUName in
+    i686)
```

Comment:
The CoreCLR script has:
```
echo "Unsupported CPU $CPUName detected, build might not succeed!"
```
for each of the non x64 options... that seems like a good thing to include here as well.